buildSrc/src/main/kotlin/configureMavenPom.kt

```diff
@@ -1,6 +1,8 @@
 import org.gradle.api.publish.maven.MavenPom
 
 fun MavenPom.configureForPbandk() {
+    name.set("Pbandk")
```

Comment:
Please review all details added to `pom.xml` thoroughly.

Maven Central mandates these attributes to be present in your `pom.xml` and I tried to choose good defaults based on your existing public language.